src/RobotHelper/src/Helper.cpp

```diff
@@ -2,8 +2,9 @@
 #include <iDynTree/Core/EigenHelpers.h>
 #include <iDynTree/yarp/YARPConversions.h>
 
-#include <RobotHelper.hpp>
-#include <Utils.hpp>
+#include <WalkingControllers/RobotHelper/Helper.h>
+#include <WalkingControllers/iDynTreeHelper/Helper.h>
+#include <WalkingControllers/YarpHelper/Helper.h>
```

Comment:
To be honest I am starting to not like the word ``helper``. It is not very descriptive and it is repeated both in the file and on the folder. What about renaming the folder/library name with something more general, allowing us to add other files in the future? What about ``utilities`` or ``interfaces``?

Reply:
`RobotHelper` could become `RobotInterface` the others `*Utilities`